cli/mender-artifact/validate.go

```diff
@@ -33,18 +33,26 @@ func validate(art io.Reader, key []byte) error {
 	// just continue checking consistency and return info if
 	// signature verification failed
 	var validationError error
+	verifiedDigitalSignature := false
+
+	// Some callers pass nil, others pass "", to indicate a missing public key.
+	isKeySpecified := (key != nil) && (len(key) > 0)
```

Comment:
Just a question, doesn't keyIsSpecified` make the if statement sound a bit better?

Reply:
I'm fine either way. `keyIsSpecified` does sound more like natural English. Beginning a boolean variable with "is" was just a holdover from a coding standard from a place I used to work. I'm happy to change it.